### ISSUE TITLE
allow to stop and emit Tensorflow graph before training method is applied

### ISSUE THREAD

**Issue by obo** [ufal/neuralmonkey#115]
Even with minimal model configuration (max_input_len=2, embedding=2, rnn=2), graph displayed in TensorBoard is too complex for browsing.

Add a command-line switch/command to just build the graph and quit, so that TensorBoard graph browser does not include Adam (or other training method) with all its gradients, etc.

**Comment by jindrahelcl** [comment]
This could be done in the neuralmonkey-check executable as well. However, I fear that the TF graph won't be less complex anyway. Btw. embedding and rnn size setting does not have an effect on the complexity of the graph.
The graph should also be simpler when using the dynamic RNN (#114).

**Comment by obo** [comment]
Solve by creating a dummy trainer (with no optimizer) and setting epochs=0 to avoid training.
No need to save the graph in this trainer, will be saved at session start.
